packages/desktop-client/src/components/reports/util.ts

```diff
@@ -18,18 +18,6 @@ export async function runAll(
   cb(data);
 }
 
-export function index<
```

Comment:
Making this strict TS compliant was a big pain.. so instead I opted to use `lodash`.